### DISCUSSION TITLE
GPU Utalization issue at Larger Models !

### DISCUSSION THREAD

**Discussion by jaifar530** [open-webui/open-webui#830] [Q&A]
Hi 

Using Ubuntu 22. 
both commands nvcc --version and nvidia-smi are showing valied outputs. 

I've noticed that the GPU is not utilized when running larger models (e.g., MiXtral8x7B, Llama 70B), yet it functions well with smaller models like Mistral and Llama 7B. Is this issue known to others, or is it just me experiencing it? By the way, I tested this on both RTX 3090 and RTX 2080, and both exhibited the same issue with the larger models.

Additionally, with the larger models (Mistral and Llama 70B), the GPU RAM is almost fully utilized, but not the GPU itself (which is very strange), while the CPU is fully utilized.

Here is the summary:

- Larger models MiXtral8x7B, Llama 70B
GPU: Not utalised
GPU RAM: utalised
CPU: utalised
RAM: Not utalised

- Small models Mistral and Llama 7B
GPU: utalised
GPU RAM: utalised
CPU: not utalised
RAM: not utalised

in summary i can use the power of GPU on small models only unfortuntly.

**Comment by justinh-rahb** [comment]
Your problem is VRAM, specifically lack of it. RTX 2080 doesn't have nearly enough for Mixtral or Llama2 70B, and the 3090 would probably only work with a 3-bit quant of Mixtral...

**Reply by codingjoe** [reply]
Be that as it may, it should still use the GPU and swap memory between the RAM and VRAM. I have 70B models running on a GPU with 16GB VRAM. Yet, I can confirm, that open-webui:ollama doesn't work for me either. My Ollama install on a separate Ubuntu VM works fine.
